GVRf/Framework/jni/engine/renderer/renderer.cpp

```diff
@@ -569,7 +569,6 @@ bool Renderer::isShader3d(const Material* curr_material) {
     case Material::ShaderType::TEXTURE_SHADER:
     case Material::ShaderType::EXTERNAL_RENDERER_SHADER:
     case Material::ShaderType::ASSIMP_SHADER:
```

Comment:
I followed the instructions for doing the rebase. Do I need to do the pull request over again or am I good?

-Nola

From: thomasflynn [mailto:notifications@github.com]
Sent: Thursday, March 3, 2016 2:09 PM
To: Samsung/GearVRf <GearVRf@noreply.github.com>
Cc: Nola Donato <nola.donato@samsung.com>
Subject: Re: [GearVRf] Fix crash in monoscopic view applications. The first problem was a ze… (#442)


In GVRf/Framework/jni/engine/renderer/renderer.cpp<https://github.com/Samsung/GearVRf/pull/442#discussion_r54956947>:

> @@ -569,7 +569,6 @@ bool Renderer::isShader3d(const Material* curr_material) {

>      case Material::ShaderType::TEXTURE_SHADER:

>      case Material::ShaderType::EXTERNAL_RENDERER_SHADER:

>      case Material::ShaderType::ASSIMP_SHADER:

looks like you need to rebase. seems you're missing the lightmap shader that was added recently.

—
Reply to this email directly or view it on GitHub<https://github.com/Samsung/GearVRf/pull/442/files#r54956947>.